src/frontend/src/routes/(new-styling)/+page.svelte

```diff
@@ -78,12 +79,21 @@
             <ul class="contents">
               {#each lastUsedIdentities as identity}
                 <li class="contents">
-                  <ButtonCard onclick={() => handleContinueAs(identity)}>
+                  <ButtonCard
+                    onclick={() => handleContinueAs(identity)}
+                    disabled={nonNullish(
+                      authLastUsedFlow.authenticatingIdentity,
+                    )}
+                  >
                     <Avatar size="sm">
-                      <UserIcon size="1.25rem" />
+                      {#if identity.identityNumber === authLastUsedFlow.authenticatingIdentity}
+                        <ProgressRing />
+                      {:else}
+                        <UserIcon size="1.25rem" />
+                      {/if}
                     </Avatar>
                     <div class="flex flex-col text-left text-sm">
-                      <div class="text-text-primary font-semibold">
+                      <div class="font-semibold">
```

Comment:
The primary color is set from the Button.

If applied here, the disabled text color was not being applied.